test/models/listing/listing.js

```diff
@@ -198,83 +207,86 @@ describe('the Listing model', () => {
     const valErr = listing.validationError;
 
     expect(valErr && valErr.termsAndConditions &&
-      !!valErr.termsAndConditions.length || false).to.equal(true);
+        !!valErr.termsAndConditions.length || false).to.equal(true);
   });
 
   it('fails validation if, for a physical good, at least one shipping options is not provided',
-    () => {
-      const listing = new Listing();
+      () => {
+        const listing = new Listing();
 
-      listing.get('metadata').set('contractType', 'PHYSICAL_GOOD');
+        listing.get('metadata').set('contractType', 'PHYSICAL_GOOD');
 
-      listing.set({
-        shippingOptions: [],
-      }, { validate: true });
+        listing.set({
+          shippingOptions: [],
+        }, { validate: true });
 
-      const valErr = listing.validationError;
+        const valErr = listing.validationError;
 
-      expect(valErr && valErr.shippingOptions &&
-        !!valErr.shippingOptions.length || false).to.equal(true);
-    });
+        expect(valErr && valErr.shippingOptions &&
+            !!valErr.shippingOptions.length || false).to.equal(true);
+      });
 
   it('fails validation if the coupon count exceeds the maximum allowable amount',
-    () => {
-      const listing = new Listing();
-      const couponData = [];
+      () => {
+        const listing = new Listing();
+        const couponData = [];
 
-      for (let i = 0; i < (listing.max.couponCount + 1); i++) {
-        couponData.push({
-          discountCode: Date.now() + Math.random(),
-          percentDiscount: 10,
-        });
-      }
+        for (let i = 0; i < (listing.max.couponCount + 1); i++) {
+          couponData.push({
+            discountCode: Date.now() + Math.random(),
+            percentDiscount: 10,
+          });
+        }
 
-      listing.set({
-        coupons: couponData,
-      }, { validate: true });
+        listing.set({
+          coupons: couponData,
+        }, { validate: true });
 
-      const valErr = listing.validationError;
+        const valErr = listing.validationError;
 
-      expect(valErr && valErr.coupons &&
-        !!valErr.coupons.length || false).to.equal(true);
-    });
+        expect(valErr && valErr.coupons &&
+            !!valErr.coupons.length || false).to.equal(true);
+      });
 
   it('fails validation if a coupon price exceeds the listing price',
-    () => {
-      const listing = new Listing();
+      () => {
+        const listing = new Listing();
 
-      listing.set({
-        item: {
-          price: 500,
-        },
-        coupons: [
-          {
-            discountCode: Date.now() + Math.random(),
-            priceDiscount: 500, // should not fail validation
+        listing.set({
+          item: {
+            price: 500,
           },
-          {
```

Comment:
I don't understand what's happening here? Is spacing really resulting in such a dramatic diff?